Sources/NiceComponents/Theme/TypeTheme.swift

```diff
@@ -12,11 +12,17 @@ public struct TypeTheme {
         public let name: String?
         public let weight: Font.Weight?
         public let size: CGFloat
+        public var dynamicTypeMaxSize: DynamicTypeSize?
 
-        public init(_ name: String? = nil, size: CGFloat, weight: Font.Weight? = nil) {
+        public init(
+                _ name: String? = nil,
+                size: CGFloat,
+                weight: Font.Weight? = nil,
+                dynamicTypeMaxSize: DynamicTypeSize? = nil) {
```

Comment:
Fix formatting here

Reply:
Not sure what formatting you're referring to here

Reply:
Everything is super indented just need to put the closing parentheses on a new line